pkg/nuclide-outline-view/lib/main.js

```diff
@@ -40,6 +41,27 @@ export type OutlineTreeForUi = {
   highlighted: boolean,
 };
 
+// Kind of outline tree - matches the names from the Language Server Protocol v2.
+export type OutlineTreeKind =
```

Comment:
It looks like this should be moved to `rpc-types.js` so you can actually use it from both `OutlineTreeForUi` and `OutlineTree`. Or is there something blocking that?